configs/samples/pjsip.conf.sample

```diff
@@ -660,6 +660,7 @@
                         ; "auth_username": Identify by the Authorization username and realm
                         ; "ip": Identify by the source IP address
                         ; "header": Identify by a configured SIP header value.
+                        ; "request": Identify by the configured SIP request line value.
```

Comment:
request_line instead of request, request has a specific meaning in SIP which extends beyond the request line

Reply:
it's request_uri now.